### DISCUSSION TITLE
Display Video as URL in Readme

### DISCUSSION THREAD

**Discussion by diamant3** [community/community#44061] [Mobile]
Video does not displaying correctly in Readme

![Screenshot_20230112_115357.jpg](https://user-images.githubusercontent.com/71203851/211972317-8b2c8174-6e21-48e1-811a-30ad3088d180.jpg)

Do i need to have a video file in my repo then link to Readme or is this a  bug?

**Comment by Dexters-Hub** [comment]
Hi, @diamant3 👋🏻 

There isn't any problem viewing it on my 📱  mobile device. To add a video, Just “ Drag & Drop” or upload ⬆️ the file to README.md and confirm the successful completion ✅  of the upload process to ensure proper functionality. 😄  There is no need to link anything. Also, Keep in mind the maximum file size is 10MB for images and gifs. 10MB for videos on a free GitHub plan. 100MB for videos on a paid GitHub plan.

**Reply by diamant3** [reply]
Okay, thanks, but that's what I do, I drag and drop the video into the readme file because I thought it would be okay for any screen size, but when I check on my phone, that's the result.😢
